PYME/Acquire/Hardware/ueye.py

```diff
@@ -534,3 +534,10 @@ def _GetDeviceInfo(self):
         self.check_success(ueye.is_DeviceInfo(self.h,ueye.IS_DEVICE_INFO_CMD_GET_DEVICE_INFO,
                                               dev_info,ueye.sizeof(dev_info)))
         return dev_info
+
+class MultiviewUEye(MultiviewCameraMixin, UEyeCamera):
```

Comment:
```suggestion
#TODO - replace MultiviewCameraMixin with a Multiview wrapper so that we don't need to #have explicit multiview versions of all cameras.
class MultiviewUEye(MultiviewCameraMixin, UEyeCamera):
```